src/builder/QuarkBuilder.sol

```diff
@@ -178,7 +186,11 @@ contract QuarkBuilder {
             }
 
             if (amountLeftToBridge > 0) {
-                revert FundsUnavailable();
+                revert FundsUnavailable(
+                    transferIntent.amount - balanceOnDstChain,
+                    transferIntent.amount - amountLeftToBridge,
+                    amountLeftToBridge
+                );
```

Comment:
Just a heads up I changed this custom error in my bridging logic refactor: https://github.com/compound-finance/quark-scripts/pull/30/files#diff-cd5318530a09b8df7f1d19dcd7fa22875ac1c8b9a51298e8f7b4e6824cf89b83R33

Nothing for you to do, but rather just letting you know how this might evolve.

Reply:
Oh yeah I saw that too 😄 
I modified it to use the same exact naming as well. 

But I found it might be useful to leave in the `missingAmount`  so that client don't have to do `requestAmount - actualAmount` when they sees errors.